packages/crayons-core/src/components/icon/icon-utils.ts

```diff
@@ -0,0 +1,102 @@
+export function registerIconlibs(items): any {
+  if (
+    Object.keys(items).includes('crayons') ||
+    Object.keys(items).includes('system')
+  ) {
+    console.error("IconLib 'crayons'|'system' cannot be registered from App.");
+    delete items['crayons'];
+    delete items['system'];
+  }
+
+  const iconlibListElem = document.createElement('ul');
+  iconlibListElem.setAttribute('id', 'fw-icon-lib-ul-list');
+  iconlibListElem.style.display = 'none';
+
+  let children = '';
+  for (const key in items) {
+    children = children + `<li id=${key}>${items[key]}</li>`;
+  }
+
+  iconlibListElem.innerHTML = children;
+  const firstBodyEle = document.body.firstChild;
+  firstBodyEle.parentNode.insertBefore(iconlibListElem, firstBodyEle);
```

Comment:
Hi, how is this different from using window?. This can also lead to collision of the keys